src/parser/jobs/pld/modules/FightOrFlight.tsx

```diff
@@ -60,18 +62,42 @@ export class FightOrFlight extends BuffWindow {
 
 		this.addEvaluator(new ExpectedActionsEvaluator({
 			expectedActions: [
-				{action: this.data.actions.GORING_BLADE, expectedPerWindow: 1},
-				{action: this.data.actions.CONFITEOR, expectedPerWindow: 1},
-				{action: this.data.actions.BLADE_OF_FAITH, expectedPerWindow: 1},
-				{action: this.data.actions.BLADE_OF_TRUTH, expectedPerWindow: 1},
-				{action: this.data.actions.BLADE_OF_VALOR, expectedPerWindow: 1},
-				{action: this.data.actions.HOLY_SPIRIT, expectedPerWindow: 1},
+				{action: actions.GORING_BLADE, expectedPerWindow: 1},
+				{action: actions.CONFITEOR, expectedPerWindow: 1},
+				{action: actions.BLADE_OF_FAITH, expectedPerWindow: 1},
+				{action: actions.BLADE_OF_TRUTH, expectedPerWindow: 1},
+				{action: actions.BLADE_OF_VALOR, expectedPerWindow: 1},
 			],
-			suggestionIcon: this.data.actions.FIGHT_OR_FLIGHT.icon,
+			suggestionIcon: actions.FIGHT_OR_FLIGHT.icon,
+			suggestionContent: <Trans id="pld.fightorflight.suggestions.gcd_actions.content">
+				Try to land at least one cast of <DataLink action="GORING_BLADE" />
+				, <DataLink action="CONFITEOR" />, <DataLink action="BLADE_OF_FAITH" />, <DataLink action="BLADE_OF_TRUTH" />
+				, <DataLink action="BLADE_OF_VALOR" />, and the highest potency combination of <DataLink action="SEPULCHRE" />, <DataLink status="DIVINE_MIGHT" />
+				empowered <DataLink action="HOLY_SPIRIT" />, <DataLink action="SUPPLICATION" />, <DataLink action="ATONEMENT" /> and <DataLink action="ROYAL_AUTHORITY" />
+				during every <DataLink action="FIGHT_OR_FLIGHT" /> window.
+			</Trans>,
+			suggestionWindowName,
+			severityTiers: SEVERITIES.MISSED_ACTIONS,
+		}))
+
+		this.addEvaluator(new ExpectedActionGroupsEvaluator({
+			expectedActionGroups: [{
+				actions: [
+					actions.ROYAL_AUTHORITY,
+					actions.ATONEMENT,
+					actions.SUPPLICATION,
+					actions.SEPULCHRE,
+					actions.HOLY_SPIRIT,
+				],
+				overrideHeader: actions.SEPULCHRE,
+				expectedPerWindow: 3,
+			}],
+			suggestionIcon: actions.FIGHT_OR_FLIGHT.icon,
 			suggestionContent: <Trans id="pld.fightorflight.suggestions.gcd_actions.content">
 				Try to land at least one cast of <DataLink action="GORING_BLADE" />
 				, <DataLink action="CONFITEOR" />, <DataLink action="BLADE_OF_FAITH" />, <DataLink action="BLADE_OF_TRUTH" />
-				, <DataLink action="BLADE_OF_VALOR" />, and a <DataLink status="DIVINE_MIGHT" /> empowered <DataLink action="HOLY_SPIRIT" />
+				, <DataLink action="BLADE_OF_VALOR" />, and the highest potency combination of <DataLink action="SEPULCHRE" />, <DataLink status="DIVINE_MIGHT" />
+				empowered <DataLink action="HOLY_SPIRIT" />, <DataLink action="SUPPLICATION" />, <DataLink action="ATONEMENT" /> and <DataLink action="ROYAL_AUTHORITY" />
 				during every <DataLink action="FIGHT_OR_FLIGHT" /> window.
```

Comment:
Playing optimally, it's impossible to get an empowered Holy Spirit and Sepulchre every FoF window. When hitting a striking dummy (bursting after 3 gcds), the 5th minute rotation restricts you from using Sepulchre unless you cut an empowered Holy Spirit or Sepulchre earlier which isn't good.

I believe the FoF window from a simplicity standpoint should have 3 of the following actions (at it's simplest level): empowered Holy Spirit, Royal Authority, Atonement, Supplication, and/or Supulchre. I believe it's confusing to the user to just use the Supulchre as an icon and show 3 uses of it. It doesn't line up with what's expected.

Reply:
Fair point. I've removed the overrideHeader icon of Sepulchre and adjusted the PR description accordingly. The column is just a bit crowded now. I agree that clarity is more important, however.